test/contrib/sge_test.py

```diff
@@ -21,9 +21,10 @@
 from glob import glob
 import unittest
 import logging
+from mock import patch
 
 import luigi
-from luigi.contrib.sge import SGEJobTask, _parse_qstat_state, _clean_task_id
```

Comment:
@Tarrasch it surprised me that this task can not work on my laptop, I found out the `_clean_task_id` is not there anymore so I just remove it in this test case.

FYI the command I ran the test on my lap top is
```
tox -e py-nonhdfs -- -s --tests=test/contrib/sge_test.py
```
let me know if I'm wrong